charts/rucio-server/values.yaml

```diff
@@ -125,6 +125,14 @@ ftsRenewal:
   voms: "cms:/cms/Role=production"
   servers: "https://fts3-devel.cern.ch:8446,https://cmsfts3.fnal.gov:8446,https://fts3.cern.ch:8446,https://lcgfts3.gridpp.rl.ac.uk:8446,https://fts3-pilot.cern.ch:8446"
 
+ftsRenewalResources:
```

Comment:
i think it would be better to put this under `ftsRenewal` instead of creating a new section `ftsRenewalResources`. I realize  that `metricsExporterResources` already exists, but I think it was a bad naming initially and we shouldn't add more of these.